Scripts/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#set -x # DEBUG
+
+# launch the unit tests suite.
+# run this script like this: ./Scripts/test.sh
+
+RDBASE=$PWD
+PYTHONPATH=$RDBASE
+LD_LIBRARY_PATH=$RDBASE/lib:$LD_LIBRARY_PATH
+
+# run them in parallel
+nprocs=`getconf _NPROCESSORS_ONLN`
+
+cd build && ctest --output-on-failure -j ${nprocs}
```

Comment:
I think the name of the build directory should be controllable from the command line. Having `build` be the default makes sense, but many of us use different names.